### DISCUSSION TITLE
Application Error: A Client-Side Exception Has Occurred

### DISCUSSION THREAD

**Discussion by ES484** [vercel/next.js#46751] [Help]
### Summary

after build nextjs website to vercel I got this error when navigate to any page: Application Error: A Client-Side Exception Has Occurred
"next": "^13.0.3",
"react": "^18.2.0",
can you help me to solve it

**Comment by icyJoseph** [comment]
Have you got any more information? Reproduction demo? How's the application setup? Do you use `_app`?
